src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +50,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
+  }
+
+  /**
+   * Loads a map object that currently displays a hardcoded location.
+   * @param {Object} coordinates - the LatLng object of user's current location
```

Comment:
`@param {!Object=}`

the `!` indicates that it can't be null (either `!` or `?` is required for anything thats not string, number, or boolean). The `=` indicates that it has a default param, so undefined can be used.

` @param {!google.maps.LatLng=}` would be even better as it would be a more specific type

Reply:
Good idea! Fixed